certified-connectors/Lex Power Sign/readme.md

```diff
@@ -0,0 +1,79 @@
+# Lex Power Sign connector
+
```

Comment:
Please update Readme.md file as per the template here 
https://github.com/microsoft/PowerPlatformConnectors/blob/3ac7cb2ca3a494950d3a6810d2aeeb69f7ccc507/templates/certified-connectors/apiProperties.json

Reply:
Hello @vmanoharas . I updated the Readme.md file to conform to the template. Regards.